ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml

```diff
@@ -22,7 +22,7 @@ resources:
       memory: 200Mi
 tests:
 - as: perfscale-rosa-multiaz-control-plane-24nodes-periodic
-  cron: 0 8 * * 4
+  cron: 0 9 */14 * 4
```

Comment:
Same note in [openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.12-nightly-x86.yaml](https://github.com/openshift/release/pull/44993/files#diff-3055446043e701a0dc2bd8aeab2b4f1f0548130387c54b2493229822c8b1c07a)